docs/ai/audits/archive/shadcn-ui-audit-2026-04-16.md

```
@@ -1,3 +1,6 @@
+> **ARCHIVED (2026-06):** Historical audit from the Radix era. The UI layer
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)